Xamarin.Forms.Platform.Android/FastRenderers/ButtonRenderer.cs

```diff
@@ -315,12 +315,12 @@ void Initialize()
 			AddOnAttachStateChangeListener(this);
 			OnFocusChangeListener = this;
 
-			Tag = this; 
-        }
+			Tag = this;
+		}
 
 		void UpdateBitmap()
 		{
-			if (Element == null)
+			if (Element == null || _isDisposed)
```

Comment:
can't we do the check at a single place, in OnPropertyChanged ? and do it on the base class, so it applies to all controls ?